packages/collector/test/tracing/messaging/bull/test.js

```diff
@@ -169,12 +165,6 @@ mochaSuiteFn.only('tracing/messaging/bull', function () {
               response,
               apiPath,
               testId,
-              // 1 x node.http.server 1
-              // 1 x bull receive
-              // 1 x otel (?)
-              // 1 x node.http.client (?)
-              // 1 x redis
-              // 1 x bull sender
```

Comment:
@abhilash-sivan Could we keep the comment please? I would love to know why there is 1 otel and one node.http.client span?

Reply:
Yeah. Added back the comment.
I think this will be resolved with https://jsw.ibm.com/browse/INSTA-15029

Reply:
Ok agree 👍